java-frontend/src/main/java/org/sonar/java/model/VisitorsBridge.java

```diff
@@ -119,14 +124,39 @@ public void visitFile(@Nullable Tree parsedTree) {
       new SymbolicExecutionVisitor(executableScanners, behaviorCache).scanFile(javaFileScannerContext);
       behaviorCache.cleanup();
     }
-    for (JavaFileScanner scanner : executableScanners) {
-      scanner.scanFile(javaFileScannerContext);
-    }
+    runChecks(javaFileScannerContext);
```

Comment:
From how I see it, we are not covering here exception occurring during SE (a few lines aboves).
We should handle them as well.